README.md

```diff
@@ -1,13 +1,25 @@
-[![Build Status](https://travis-ci.org/SchibstedSpain/Barista.svg?branch=master)](https://travis-ci.org/SchibstedSpain/Barista)
-[![Download](https://api.bintray.com/packages/schibstedspain/maven/barista/images/download.svg)](https://bintray.com/schibstedspain/maven/barista/_latestVersion)
-
 # Barista
 **The guy who serves a great Espresso**
 
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg?label=Travis+CI)](https://travis-ci.org/SchibstedSpain/Barista)
+[![Download](https://api.bintray.com/packages/schibstedspain/maven/barista/images/download.svg)](https://bintray.com/schibstedspain/maven/barista/_latestVersion)
+[![Hex.pm](https://img.shields.io/hexpm/l/plug.svg)](LICENSE.md)
+
 ![Barista logo](art/barista-logo.png)
 
 Barista makes developing UI test faster, easier and more predictable. Built on top of Espresso, it provides a simple an discoverable API, removing most of the boilerplate and verbosity of common Espresso tasks. You and your Android team will write tests with no effort.
 
+ - [Download](#download)
+ - [API Overview](#api-overview)
+    - [Barista’s Interactions API](#baristas-interactions-api)
+    - [Barista’s Assertions API](#baristas-assertions-api)
```

Comment:
To make it faster to read, I would just say `Interactions` and `Assertions. 
People already know they're in Barista, and they're clearly part of `API Overview`. 
What do you think?

Reply:
Done!